### PR TITLE
dep2nix: init at 0.0.1

### PR DESCRIPTION
**Pull request by Mic92**
###### Motivation for this change

cc @qknight 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/dep2nix/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    rev = version;
+    sha256 = "05b06wgcy88fb5ccqwq3mfhrhcblr1akpxgsf44kgbdwf5nzz87g";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    description = "Convert `Gopkg.lock` files from golang dep into `deps.nix`";
+    license = licenses.bsd3;
+    homepage = https://github.com/nixcloud.io/dep2nix;
```
Comment (gebner) [inline pkgs/development/tools/dep2nix/default.nix]: I get a 404 for this URL.

Comment (Mic92) [reply pkgs/development/tools/dep2nix/default.nix]: fixed

File: pkgs/development/tools/dep2nix/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, buildGoPackage
+, makeWrapper, nix-prefetch-git }:
+
+buildGoPackage rec {
+  name = "dep2nix-${version}";
+  version = "0.0.1";
+
+  goPackagePath = "github.com/nixcloud.io/dep2nix";
```
Comment (orivej) [inline pkgs/development/tools/dep2nix/default.nix]: Please fix this too.

Comment (Mic92) [reply pkgs/development/tools/dep2nix/default.nix]: Also fixed upstream: https://github.com/nixcloud/dep2nix/pull/11

**Pull request merged**